app/src/main/java/org/oppia/android/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -17,7 +17,7 @@ class AppVersionViewModel @Inject constructor(
   context: Context
 ) : ObservableViewModel() {
 
-  private val versionName: String = context.getVersionName()
+  private val versionName: String = context.getVersionName().toString()
```

Comment:
For this & the change in `RemoteAuthNetworkInterceptor`, won't this result in "null" being the string if the version name is `null`? I also have a comment in the extensions file seeking more clarification, as well.

Reply:
I have added a fallback version name in the extensions file and reverted this change.